exporters/css/src/content/token.ts

```diff
@@ -1,20 +1,26 @@
 import { NamingHelper, CSSHelper } from "@supernovaio/export-helpers"
-import { Token, TokenGroup } from "@supernovaio/sdk-exporters"
+import { Token, TokenGroup, TokenType } from "@supernovaio/sdk-exporters"
 import { exportConfiguration } from ".."
 
 export function convertedToken(token: Token, mappedTokens: Map<string, Token>, tokenGroups: Array<TokenGroup>): string {
   // First creating the name of the token, using helper function which turns any token name / path into a valid variable name
   const name = tokenVariableName(token, tokenGroups)
 
   // Then creating the value of the token, using another helper function
-  const value = CSSHelper.tokenToCSS(token, mappedTokens, {
+  let value: string | number = CSSHelper.tokenToCSS(token, mappedTokens, {
     allowReferences: exportConfiguration.useReferences,
     decimals: exportConfiguration.colorPrecision,
     colorFormat: exportConfiguration.colorFormat,
     tokenToVariableRef: (t) => {
       return `var(--${tokenVariableName(t, tokenGroups)})`
     },
   })
+
+  if (exportConfiguration.useFontWeightAsNumbers && token.tokenType === TokenType.fontWeight) {
+    let candidate = +(value?.replaceAll('"', ''))
```

Comment:
As we discussed, it should never return the value wrapped in quotes for font-weight, that's invalid CSS then. So the current behavior of this flag (when turned off) will lead to incorrect values

+ when i see `useFontWeightAsNumbers` flag, my expectations are that string values (such as bold, semibold, normal, etc) will be returned as 700, 600, 500 instead of the word. But it's not what this does

Here is a mapping table from OpenType specs:
![CleanShot 2024-08-25 at 11 52 48@2x](https://github.com/user-attachments/assets/6587ba4e-2249-469e-8a41-004668833a15)